src/sentry/api/endpoints/group_events.py

```diff
@@ -105,22 +104,19 @@ def _get_events_snuba(self, request, group, environments, query, tags, start, en
                 raise GroupEventsError(
                     'Boolean search operator OR and AND not allowed in this search.')
 
-        snuba_cols = SnubaEvent.minimal_columns if full else SnubaEvent.selected_columns
```

Comment:
Was this actually correct?
minimal_columns if full otherwise more columns ?

Reply:
I think it's correct. Reason is that full always needs the nodestore load anyway, otherwise we can avoid it.